src/order-basket/drug-search.ts

```diff
@@ -0,0 +1,100 @@
+import { getDrugByName } from '../api/api';
+import { getCommonMedicationByUuid } from '../api/common-medication';
+import { OrderBasketItem } from '../types/order-basket-item';
+import { daysDurationUnit } from '../constants';
+import _ from 'lodash-es';
```

Comment:
This imports the whole lodash-es package. Do we need this? Or just something specific?